### ISSUE TITLE
incorporate mason from @gaborcsardi for scaffoldWidget?

### ISSUE THREAD

**Issue by timelyportfolio** [ramnathv/htmlwidgets#124]
Based on my understanding of the new `mason` from @gaborcsardi, we could easily create a template and make `htmlwidget` package creation one step rather than the current two of create package and then scaffoldWidget.

**Comment by gaborcsardi** [comment]
Yeah, I could have a template that creates the package and just calls htmlwidgets:: scaffoldWidget.

**Comment by ramnathv** [comment]
Nice. This could be very useful!
